test/test.js

```diff
@@ -121,6 +121,104 @@ describe('suitcss', function() {
       });
     });
 
+    describe('using the transform option in postcss-import', function() {
+      it('should use a default transform function that just returns the css', function(done) {
+        suitcss('@import "./util.css";', {
+          lint: true,
+          root: 'test/fixtures'
+        }).then(function(result) {
+          expect(result.css).to.equal('.u-img {\n  border-radius: 50%;\n}');
+          done();
+        })
+        .catch(function(err) {
+          done(err);
+        });
+      });
+
+      it('should call a custom transform function with the imported component', function(done) {
+        var transformStub = sinon.stub().returns('body { color: blue; }');
+
+        suitcss('@import "./util.css";', {
+          lint: true,
+          root: 'test/fixtures',
+          'postcss-easy-import': {
+            transform: transformStub
+          }
+        }).then(function(result) {
+          expect(transformStub.calledOnce).to.be.true;
+          expect(transformStub.getCall(0).args[0]).to.equal('.u-img {\n  border-radius: 50%;\n}\n');
+          expect(result.css).to.equal('body { color: blue; }');
+          done();
+        })
+        .catch(function(err) {
+          done(err);
+        });
+      });
+
+      it('should also work with a promise returned from the custom transform function', function(done) {
+        suitcss('@import "./util.css";', {
+          lint: true,
+          root: 'test/fixtures',
+          'postcss-easy-import': {
+            transform: function() {
+              return Promise.resolve('body { font: red; }');
+            }
+          }
+        }).then(function(result) {
+          expect(result.css).to.equal('body { font: red; }');
+          done();
+        })
+        .catch(function(err) {
+          done(err);
+        });
+      });
+    });
+
+    describe('using the onImport option postcss-import', function() {
+      var updateWatchTaskFilesSpy, revert;
+
+      beforeEach(function() {
+        updateWatchTaskFilesSpy = sinon.spy();
+        revert = suitcss.__set__('updateWatchTaskFiles', updateWatchTaskFilesSpy);
+      });
+
+      afterEach(function() {
+        revert();
+      });
+
+      it('should call the updateWatchTaskFiles function with the file paths', function(done) {
+        suitcss('@import "./util.css";', {
+          lint: true,
+          root: 'test/fixtures'
+        }).then(function() {
+          expect(updateWatchTaskFilesSpy.getCall(0).args[0][0]).to.contain('util.css');
```

Comment:
should this *contain* `./utils.css`? 
Maybe `.to.equal(['./utils.css'])` or is `updateWatchTaskFilesSpy` called with a single filename?